bundles/AdminBundle/config/pimcore/two_factor_authentication.yaml

```diff
@@ -1,25 +1,10 @@
-services:
-    _defaults:
-        autowire: true
-        autoconfigure: true
-        public: false
-
-    Pimcore\Bundle\AdminBundle\EventListener\TwoFactorListener:
```

Comment:
we still need to register the event listener for `onAuthenticationAttempt`..right?

Reply:
is not needed, I refactored the 2fa a bit. I am still not happy with how it is implemented. If 2fa is required, you still get logged into the backend and can remove the Overlay with 2fa or even the popup and still work in Pimcore. I tried to refactor to have a separate login screen, but it is not easy to implement... the implementation is quite tricky. I think we should leave it as is and create another PR to make that better.

Reply:
see https://github.com/pimcore/pimcore/pull/13909#issuecomment-1406683977